rpc-broker/src/policy.h

```diff
@@ -0,0 +1,88 @@
+/*
+ Copyright (c) 2018 AIS, Inc.
+
+ This program is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program; if not, write to the Free Software
+ Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+*/
+
+
+#define RULES_FILENAME "/etc/rpc-broker.rules"
+#define RULES_MAX_LENGTH 256
+#define RULE_MAX_LENGTH  512
+
+struct rule {
+    uint8_t out:1;
```

Comment:
Nice optimization, but why not use `stdbool.h`? It is a C99 thing though, so kinda contradicting other comments, but you are using `bool` in other places already! :)